elixir/apps/web/lib/web/live/actors/show.ex

```diff
@@ -223,7 +223,7 @@ defmodule Web.Actors.Show do
             </.link>
           </:col>
           <:empty>
-            <div class="text-center text-neutral-500 p-4">No authorizations to display</div>
+            <div class="text-center text-neutral-500 p-4">No activity to display</div>
```

Comment:
We should have a better explanation of activities in an empty state:
```suggestion
            <div class="text-center text-neutral-500 p-4">No activity to display. Activities are created when the client attempts to access a resource.</div>
```

Reply:
Good call, updated!

Reply:
Added explanation to a `:help` section so it's shown even in populated tables.